.chloggen/multimod-diff-rc.yaml

```
@@ -0,0 +1,9 @@
+change_type: bug_fix
+
+component: multimod
+
+note: Changes the return code for diff to match the behaviour it is replacing
```

This does not tell much for non-Collector repositories 😉

---

What about what you put in the PR description?

```suggestion
note: Return a non-0 exit code if there are no changes to ensure the build does not proceed with the release
```

---

Also we should say that 0 exit code is returned when there are changes.